smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ClientRuntimeTypes.kt

```diff
@@ -120,8 +120,6 @@ object ClientRuntimeTypes {
         val SDKLogLevel = runtimeSymbol("SDKLogLevel")
         val ClientLogMode = runtimeSymbol("ClientLogMode")
         val IdempotencyTokenGenerator = runtimeSymbol("IdempotencyTokenGenerator")
-        val DefaultRetryStrategy = runtimeSymbol("DefaultRetryStrategy")
-        val RetryStrategyOptions = runtimeSymbol("RetryStrategyOptions")
         val DefaultRetryErrorInfoProvider = runtimeSymbol("DefaultRetryErrorInfoProvider")
```

Comment:
Does this need to be moved too? I don't see `DefaultRetryErrorInfoProvider` in the new `SmithyRetries` module actually, is that intentional?

Reply:
@milesziemer `DefaultRetryErrorInfoProvider` stayed in `ClientRuntime` because it has dependencies on things like service errors, HTTP requests, etc.

If I moved it, `SmithyRetries` would have to depend on `ClientRuntime`, and that would create a circular dependency (a bad practice in any language but a compile error in Swift.)  Perhaps the error info provider can be moved out later when its dependencies themselves are modularized.